Dockerfile

```diff
@@ -186,9 +186,9 @@ RUN ./contrib/download-frozen-image-v2.sh /docker-frozen-images \
 # Download man page generator
 RUN set -x \
 	&& export GOPATH="$(mktemp -d)" \
-	&& git clone -b v1.0.3 https://github.com/cpuguy83/go-md2man.git "$GOPATH/src/github.com/cpuguy83/go-md2man" \
-	&& git clone -b v1.2 https://github.com/russross/blackfriday.git "$GOPATH/src/github.com/russross/blackfriday" \
-	&& go get -v -d github.com/cpuguy83/go-md2man \
+	&& git clone -b v1.0.4 https://github.com/cpuguy83/go-md2man.git "$GOPATH/src/github.com/cpuguy83/go-md2man" \
+	&& git clone -b v1.4 https://github.com/russross/blackfriday.git "$GOPATH/src/github.com/russross/blackfriday" \
+	&& go get -v -d github.com/russross/blackfriday \
```

Comment:
(it took me awhile to track down the go.net is why)

On Tue, Dec 1, 2015 at 3:48 PM, Jessie Frazelle <jess@docker.com> wrote:

> I did that because go-md2man only has 1 dep and its blackfriday, i was
> just making that clearer, blackfriday has a few deps
>
> On Tue, Dec 1, 2015 at 3:44 PM, Alexander Morozov <
> notifications@github.com> wrote:
>
>> In Dockerfile
>> <https://github.com/docker/docker/pull/18352#discussion_r46357564>:
>>
>> > @@ -186,9 +186,9 @@ RUN ./contrib/download-frozen-image-v2.sh /docker-frozen-images \
>> >  # Download man page generator
>> >  RUN set -x \
>> >  	&& export GOPATH="$(mktemp -d)" \
>> > -	&& git clone -b v1.0.3 https://github.com/cpuguy83/go-md2man.git "$GOPATH/src/github.com/cpuguy83/go-md2man" \
>> > -	&& git clone -b v1.2 https://github.com/russross/blackfriday.git "$GOPATH/src/github.com/russross/blackfriday" \
>> > -	&& go get -v -d github.com/cpuguy83/go-md2man \
>> > +	&& git clone -b v1.0.4 https://github.com/cpuguy83/go-md2man.git "$GOPATH/src/github.com/cpuguy83/go-md2man" \
>> > +	&& git clone -b v1.4 https://github.com/russross/blackfriday.git "$GOPATH/src/github.com/russross/blackfriday" \
>> > +	&& go get -v -d github.com/russross/blackfriday \
>>
>> I wonder why this line is changed?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/docker/docker/pull/18352/files#r46357564>.
>>
>
>